### ISSUE TITLE
doesn't appear to work on fedora 21

### ISSUE THREAD

**Issue by jmccormick2001** [sasha-alias/sqltabs_tracker#2]
I downloaded the linux version, ran the executable and a blank window shows up, this on CentOS Linux release 7.1.1503 (Core)

**Comment by sasha-alias** [comment]
yep, it seems I need to make a separate build for CentOS distros due to postgresql driver linking.
thanks for reporting, I will let you know here when done
